Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -55,6 +59,15 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         XCTAssertFalse(viewController.notificationFired)
     }
 
+    #if !targetEnvironment(macCatalyst)
+    
+    func testInstantiate() {
+        let myViewController = MyViewController.instantiate(from: "TestStoryboard")
```

Comment:
Found it! When running unit tests you need to define the `Bundle`.
```suggestion
        let myViewController = MyViewController.instantiate(from: "TestStoryboard", bundle: Bundle(for: UIViewControllerExtensionsTests.self))
```